src/OrbitGl/TimeGraph.cpp

```diff
@@ -682,6 +682,9 @@ void TimeGraph::Draw(Batcher& batcher, TextRenderer& text_renderer, uint64_t cur
                      PickingMode picking_mode, float z_offset) {
   ORBIT_SCOPE("TimeGraph::Draw");
 
+  // Don't draw while loading to avoid contention with the loading thread.
+  if (app_->IsLoadingCapture()) return;
```

Comment:
I rather would prefer app_ not calling draw from TimeGraph to avoid this dependence. TimeGraph shouldn't know about if a capture is being loaded.

Reply:
I totally agree, I moved it up to CaptureWindow.